tools-local/regenerate-readme-table.proj

```diff
@@ -8,43 +8,39 @@
       Column and row configuration for the readme table. The abbreviations match up with
       reference-style Markdown links in the readme file to figure out what to put in the cells.
     -->
+    <PropertyGroup>
+      <TablePath>$([MSBuild]::NormalizePath('$(RepoRoot)', 'docs', 'project', 'dogfooding.md'))</TablePath>
+    </PropertyGroup>
+
     <ItemGroup>
-      <Branches Include="Master" Abbr="master" />
-      <Branches Include="Release/3.0.X" Abbr="3.0.X" />
-      <Branches Include="Release/2.2.X" Abbr="2.2.X" />
-      <Branches Include="Release/2.1.X" Abbr="2.1.X" />
+      <Branches Include="Master (CoreCLR)" Abbr="6.0.X-coreclr" />
```

Comment:
What is Master CoreCLR? and that branch number?

Reply:
It's the master builds using a coreclr VM. The branch number is just a moniker for the links. I'll think of a better name.